container_workflow_tool/main.py

```diff
@@ -583,13 +570,23 @@ def dist_git_changes(self, rebase: bool = False):
                 "cwt git push && cwt build"
                 "[base/core/s2i] --repo-url link-to-repo-file")
 
+    def dist_git_changes(self, rebase: bool = False):
```

Comment:
What about
```suggestion
    def dist_git_merge_changes(self, rebase: bool = False):
```

Reply:
Fixed by commit https://github.com/sclorg/container-workflow-tool/pull/71/commits/80e57ad69a6587abc547a99c6632d6d34665f2aa.

The version was also bumped because there are none-compatible changes.